### ISSUE TITLE
Error when attempting to play any Sequence in screenspace layer 

### ISSUE THREAD

**Issue by yoshisman8** [fantasycalendar/FoundryVTT-Sequencer#103]
**Describe the bug**
I'm receiving a silent (No red error notification) error when attempting to execute any sequence. This is the sequence in question:
```js
    new Sequence()
    .effect()
        .file(stages[level])
        .screenSpace()
        .screenSpacePosition({ x:475, y:300 })
        .name("LimitBreakGauge")
        .persist()
        .locally()
    .play();
```

**Screenshots**
![image](https://user-images.githubusercontent.com/4214215/191778638-2c1a4d16-a0aa-421c-bd5e-fe409d0034ff.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Foundry VTT's app

**Foundry**
 - Version: 10.286
 - System: Pathfinder 2nd Edition v4.1.2
 
**Modules**
``` 
Ace Library - 1.8.3
Actually Private Rolls - 1.1.1.1
Bar Brawl - 1.7.3
Battlezoo Ancestries: Dragons for Pathfinder 2e by Roll for Combat - 1.0.2
Boss Bar - 1.0.1
Code Mirror - 5.58.3-fvtt5
Color Picker - 1.1
Combat Carousel - 0.3.0
Combat Enhancements - 1.2.0
Dice so nice! - 4.6.2
FXMaster - 3.5.0
JB2A - Patreon Complete collection - 0.4.9
lib - Color Settings - 2.9.0
libWrapper - 1.12.10.0
Macro Editor - 1.1.2
Math.js - 7.5.1.-fvtt2
Monk's Little Details - 10.2
Multilevel Tokens -1.6.0
Party Resources - 1.3.1
PF2e Companion Compendia - 5.4.1
PF2e Dorako UI - 1.6.5
PF2e Extempore Effects - 1.3.0
PF2e Keybinds Menagerie - 4.10.1
PF2e Modifiers Matter - 1.5.1
PF2e Workbench - 4.2.1
Polyglot - 1.8.4.1
PopOut! - 2.8
Quick Insert - Search Widget - 2.7.1
Sequencer - 2.3.7
Settings Extender - 1.2.1
socketlib - 1.0.11
SortableJS - 1.14.0-fvtt1
Token Action HUD - 3.0.8
Token Magic FX - 0.6.1.2
Tokenstamp 2 for Foundry VTT - 1.2.1
Token Tooltip Alt - 3.1.4
Warp Gate - 1.14.0
```

**Comment by yoshisman8** [comment]
new development: this only affects effects happening in the Screenspace layer.

**Comment by Haxxer** [comment]
Fixed in 2.3.8
